packages/timers/src/index.ts

```diff
@@ -54,7 +46,7 @@ class DeferredTimer<T> {
     // stopped. Because channels swallow rejections which settle after stop, we
     // use this mechanism to make any pending call which has received the
     // deferred promise resolve to `{ done: true }`.
-    this.reject(new TimeoutError("THIS ERROR SHOULD NEVER BE SEEN"));
```

Comment:
There is no need to use a custom error here because it is mainly used to clear pending timeouts. No one will ever see this error. See https://channel.js.org/docs/inverted-channels.